### ISSUE TITLE
Store config in project dir

### ISSUE THREAD

**Issue by laggingreflex** [appnexus/sicksync#68]
Right now config related to all projects are stored in `~/.sicksync/config.json` combined 

I use BashOnWindows so the home `~/` in bash is not my home in windows, so getting to the bash's `./sicksync.conf.json` is sometimes tricky, and there's a duplication of configs that way.

How about if configs related to projects be stored in the project directory itself?

**Comment by joelgriffith** [comment]
Yeah, there's been talk about doing something like what other software does (look in the current directory for configs and fallback either up the fs path, defaulting to some place).

This would be a somewhat larger feature, but I'll try and give it some attention at some point.
